base/src/main/java/org/mozilla/jss/pkcs12/AuthenticatedSafes.java

```diff
@@ -185,9 +185,8 @@ public SEQUENCE getSafeContentsAt(Password password, int index)
                 SEQUENCE.OF_Template seqt = new SEQUENCE.OF_Template(
                                                     SafeBag.getTemplate() );
                 return (SEQUENCE) ASN1Util.decode(seqt, os.toByteArray());
-              } else {
-                throw e;
               }
+            throw e;
```

Comment:
This code should be indented with 2 spaces to match the original code block. Ideally, the code block could be indented with 4 spaces, but it's up to you if you want to fix it or just leave it as is.